From a.beregalov@gmail.com  Tue Jun  2 12:05:41 2009
From: Alexander Beregalov <a.beregalov@gmail.com>
Date: Wed, 13 May 2009 17:23:42 +0400
Subject: Staging: comedi: data.c should include delay.h
To: gregkh@suse.de
Message-ID: <20090513132342.GA16280@orion>
Content-Disposition: inline


Fix this build error:
.../data.c:86: error: implicit declaration of function 'udelay'

Signed-off-by: Alexander Beregalov <a.beregalov@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/kcomedilib/data.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/comedi/kcomedilib/data.c
+++ b/drivers/staging/comedi/kcomedilib/data.c
@@ -26,6 +26,7 @@
 #include "../comedidev.h"
 
 #include <linux/string.h>
+#include <linux/delay.h>
 
 int comedi_data_write(void *dev, unsigned int subdev, unsigned int chan,
 	unsigned int range, unsigned int aref, unsigned int data)
